lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-job.rb

```diff
@@ -2,6 +2,7 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_AnsibleTower_ConfigurationManager_Job < MiqAeServiceOrchestrationStack
     expose :job_template, :association => true
     expose :refresh_ems
+    expose :stdout, :method => :raw_stdout
```

Comment:
@bzwei At some point the service models are going to go away in favor of REST API, which is going to work directly with app/models instead of service models. So it would be better to keep the method names consistent, calling stdout in models and raw_stdout in service models means more changes for the user scripts.